src/styles/index.scss

```diff
@@ -1,3 +1,136 @@
+$main-accent: #060b35;
+$secondary: #616070;
+$white: #fff;
+$blue-accent: #00acdc;
+
 body {
   margin: 0;
 }
+
+.product {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  width: 200px;
+  height: 406px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: $white;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+  }
+
+  &__name {
+    width: 166px;
+    margin-top: 40px;
+    margin-bottom: 0;
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    align-self: flex-start;
+    margin-left: 16px;
```

Comment:
give padding for the card instead of this left margin. remove it everywhere

Reply:
not fixed